### PR TITLE
Add platform support for delta ag9032v2a 

### PR DESCRIPTION
**Pull request by hans-tseng**
Add platform support for delta ag9032v2a 
      CPU: Broadwell-DE
      ASIC: BMC56870 with <100Gx32 + 10Gx1>
      BMC: AST2520:
**- What I did**
     add the new platform of delta ag9032v2a 
**- How I did it**
      1.provide the QSFP and SWPLD attributes and create the virtual buses of I2C
       2. provide the port configuration 
**- How to verify it**
      1. psuutil.py
      2. sfputil.py
      3. bcmcd

**- Known issue**
     The port LED is not ready. We noticed that BCM chip had the M0 FW initialization issue while bringing up SDK with "BRCM SAI ver: [3.5.2.3], OCP SAI ver: [1.4], SDK ver: [6.5.14]"   and here is the information :
root@sonic:/home/admin# bcmcmd "M0 status"
M0 status
0:soc_iproc_data_send_wait: No response for msg 2
M0 FW is NOT Running
M0 FW Version is 0.0
Host FW Version is 1.0
Host and M0 FW Versions do not match!!!

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/\ [hunk 1]
```diff
@@ -0,0 +1,1809 @@
+#include <linux/init.h>
+#include <linux/slab.h>
+#include <linux/i2c/pca954x.h>
+#include <linux/i2c-mux.h>
+#include <linux/device.h>
+#include <linux/module.h>
+#include <linux/kernel.h>
+#include <linux/sysfs.h>
+#include <linux/platform_device.h>
+#include <linux/i2c.h>
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/\]: what is this filename?

**Comment by lguohan** [comment]
> CodeFlow for GitHub does not download executable files. If you trust this file and would like to run it, please download it yourself through GitHub.

cannot have swp file in the repo.

---
Refers to: platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/.delta_ag9032v2_platform.c.swp:1 in 03dbb54. [](commit_id = 03dbb54c4c14d213ddf6bb84f57840c92939a343, deletion_comment = False)

**Comment by lguohan** [comment]
please remove minigraph.xml, not needed.

File: device/delta/x86_64-delta_ag9032v2a-r0/sensors.conf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# libsensors configuration file for AG9032v2a
+# ------------------------------------------------
+#
+
```
Comment (lguohan) [inline device/delta/x86_64-delta_ag9032v2a-r0/sensors.conf]: how come this file is empty?

File: platform/broadcom/one-image.mk [hunk 3]
```diff
@@ -28,7 +28,6 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
                                $(ACCTON_AS5812_54X_PLATFORM_MODULE) \
                                $(ACCTON_AS5835_54X_PLATFORM_MODULE) \
                                $(ACCTON_AS9716_32D_PLATFORM_MODULE) \
-                               $(ACCTON_AS5835_54T_PLATFORM_MODULE) \
```
Comment (lguohan) [inline platform/broadcom/one-image.mk]: why do you remove ACCTON sku?

**Comment by lguohan** [comment]
there are many mistakes on this PR. Please be more careful when you submitting PR.

File: platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/\ (hunk 1 above)
Comment (hans-tseng) [reply platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/\]: The "platform_device.h" is the kernel file and we need it to create the attribute for SWPLD

File: device/delta/x86_64-delta_ag9032v2a-r0/sensors.conf (hunk 2 above)
Comment (hans-tseng) [reply device/delta/x86_64-delta_ag9032v2a-r0/sensors.conf]: Those devices is handled by BMC and this file is not necessary. I will remove it

**Comment by hans-tseng** [comment]
Hi sir:
   I updated the file which you noticed and thanks to your kindly mention

File: platform/broadcom/one-image.mk [hunk 4]
```diff
@@ -28,7 +28,7 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
...
+			       $(ACCTON_AS5835_54T_PLATFORM_MODULE) \
```
Comment (lguohan) [inline platform/broadcom/one-image.mk]: you are still modifying this line, do not modify it.

File: platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/\ (hunk 1 above)
Comment (lguohan) [reply platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/\]: you do not understand my question. this filename '\' is not correct.

**Comment by lguohan** [comment]
please resolve conflict.

File: platform/broadcom/one-image.mk (hunk 4 above)
Comment (hans-tseng) [reply platform/broadcom/one-image.mk]: Hi sir :
   I fixed the conflict file

**Comment by lguohan** [comment]
I see see this file exists "platform/broadcom/sonic-platform-modules-delta/ag9032v2a/modules/\ "

please fix.

**Pull request merged**